code/datums/perks/fate.dm

```diff
@@ -147,77 +139,78 @@
 		final_oddity += stat
 		final_oddity[stat] = rand(1,7)
 	W.AddComponent(/datum/component/inspiration, final_oddity)
-	W.AddComponent(/datum/component/atom_sanity, 1, "")
+	W.AddComponent(/datum/component/atom_sanity, 1, "") //sanity gain by area
+	W.sanity_damage -= 1 //damage by view
```

Comment:
It's probably better be done by components, like it's done with cyberplants and oddities. I'm not sure that var even affects area.

Reply:
this var affects whoever sees the item, the component it owns affects the area.

Reply:
this var affects whoever sees the item, the component it owns affects the area.